extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/SessionsInfinispanConfig.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.time.Duration;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Configuration of Vert.x Web sessions stored in remote Infinispan cache.
+ */
+@ConfigGroup
+public class SessionsInfinispanConfig {
+    /**
+     * Name of the Infinispan cache used to store session data. If it does not exist, it is created
+     * automatically from Infinispan's default template {@code DIST_SYNC}.
+     */
+    @ConfigItem(defaultValue = "vertx-web.sessions")
```

Comment:
I'm wondering if we need to make it vert.x-web, and more quarkus.sessions? No strong opinions.

Reply:
I mostly just followed Vert.x Web defaults, the only place I didn't was the name of the session cookie. It probably makes sense to change this to `quarkus.sessions`.